// Protocol buffer descriptor for ISIS ControlNet class hierarchy
//
// 2017-12-11 Kristin Berry - Separated top-level messages into different files
// 2018-06-13 Jesse Mapel - Added flag to indicate it is a protobuf 2 file

syntax="proto2";

package Isis;

message ControlNetLogDataProtoV0001 {
  message Point {
    message Measure {
      message DataEntry {
        required int32 dataType = 2;
        required double dataValue = 3;
      }

      repeated DataEntry loggedMeasureData = 2;
    }

    repeated Measure measures = 2;
  }

  repeated Point points = 1;
}
